### PR TITLE
SNOW-903604 Zombie dbus processes generated by one of our dependencies github.com/99designs/keyring

### PR DESCRIPTION
**Pull request by sfc-gh-dszmolka**
SNOW-903604 Zombie dbus processes generated by one of our dependencies github.com/99designs/keyring

After long consideration and discussion how we want to address the issue we decided to go with only documenting for now, which is what i tried to do here, alongside with trying to detect if the OS is susceptible for the issue (not all Linux are)

If so, log a warning of the issue + workaround which is the same as documented in the README now.

Of course this is by no means a solution - long term solution will be to possibly replace the current keyring implementation.

### Description
Please explain the changes you made here.

### Checklist
- [ ] Code compiles correctly
- [ ] Run ``make fmt`` to fix inconsistent formats
- [ ] Run ``make lint`` to get lint errors and fix all of them
- [ ] Created tests which fail without the change (if possible)
- [ ] All tests passing
- [ ] Extended the README / documentation, if necessary

### CODE REVIEW

File: util.go [hunk 1]
```diff
@@ -292,3 +293,30 @@ func contains[T comparable](s []T, e T) bool {
...
+}
+
+func canDbusLeakProcesses() (bool, string) {
+	// TODO: delete this once we replaced 99designs/keyring (SNOW-1017659) and/or keyring#103 is resolved
+	leak := false
+	message := ""
+
+	valDbus, haveDbus := os.LookupEnv("DBUS_SESSION_BUS_ADDRESS")
+	if !haveDbus || strings.Contains(valDbus, "unix:abstract") {
+		// if DBUS_SESSION_BUS_ADDRESS is not set or set to an abstract socket, it's not necessarily a problem, only
```
Comment (sfc-gh-pfus) [inline util.go:313]: Is there some comment missing?

Comment (sfc-gh-dszmolka) [reply util.go:313]: no, i was considering lines 313-314 together 😅  , comment meaning ... 'not necessarily a problem, only if the dbus-daemon is running'

Comment (sfc-gh-pfus) [reply util.go:313]: I'd prefer having full sentences, not joining comments with code :)

Comment (sfc-gh-dszmolka) [reply util.go]: fixed!

**Review by sfc-gh-pfus — APPROVED** [review]

**Pull request merged**